frontend/src/components/form/base/BasePicker.vue

```diff
@@ -175,7 +176,8 @@ export default {
       } else {
         this.setValueOfPicker(val)
       }
-    }
+    },
+    debounce
```

Comment:
Remove this, you can use the imported `debounce` without adding it as a method. If it needed to be a method, you would have to call it using `this.debounce` anyways.